src/main/java/Duke.java

```diff
@@ -18,7 +18,7 @@ public class Duke {
 
     private final static int CAPACITY = 100;
```

Comment:
I think the order should have been `private static final` instead.

Reply:
Will take note.